### PR TITLE
Bump revision to 3.2.7-3

### PR DESCRIPTION
**Pull request by chipironcin**

### CODE REVIEW

**Comment by prydonius** [comment]
Looks like some replicaSet tests are failing. Can you take a look?

**Comment by chipironcin** [comment]
Of course, I will take care of this!

File: README.md [hunk 1]
```diff
@@ -123,8 +123,8 @@ Inside `myapp`, use `mongodb` as the hostname for the MongoDB server.
 
 A [replica set](https://docs.mongodb.com/manual/replication/) can easily be setup with the Bitnami MongoDB Docker Image using the following environment variables:
 
- - `MONGODB_REPLICASET_MODE`: MongoDB replica set mode. Possible values `primary`,`secondary` or `arbiter`. No defaults.
- - `MONGODB_REPLICASET_NAME`: MongoDB replica set name. Defaults to `replicaset`.
+ - `MONGODB_REPLICA_SET_MODE`: MongoDB replica set mode. Possible values `primary`,`secondary` or `arbiter`. No defaults.
```
Comment (sameersbn) [inline README.md]: Any particular reason for the renaming?

Comment (chipironcin) [inline README.md]: Yes, even they are being mapped in app.entrypoint.sh, naming doe snot follow the rule for naming env vars in Bitnami Containers and Nami utility. Each word should be separated by underscore.

**Comment by chipironcin** [comment]
test this please

**Comment by chipironcin** [comment]
test this please

**Pull request merged**